packages/react-ui-core/src/AutoSuggestField/AutoSuggestField.js

```diff
@@ -81,6 +81,14 @@ export default class AutoSuggestField extends Component {
     document.addEventListener('keydown', this.onKeyDown)
   }
 
+  componentWillReceiveProps(nextProps) {
+    if (!(this.props.value === nextProps.value)) {
```

Comment:
I would write a test to check the state once the props are changed.

Reply:
Added